src/java.base/unix/classes/sun/nio/fs/UnixFileSystemProvider.java

```diff
@@ -281,10 +300,10 @@ public void checkAccess(Path obj, AccessMode... modes) throws IOException {
         } else {
             for (AccessMode mode: modes) {
                 switch (mode) {
-                    case READ : r = true; break;
-                    case WRITE : w = true; break;
-                    case EXECUTE : x = true; break;
-                    default: throw new AssertionError("Should not get here");
+                    case READ -> r = true;
+                    case WRITE -> w = true;
+                    case EXECUTE -> x = true;
+                    default -> throw new AssertionError("Should not get here");
```

Comment:
The switch should be exhaustive so I assume the default case is not required here.

Reply:
Addressed in next push